conf/core/ConfigOptions.asciidoc

```diff
@@ -1855,6 +1880,58 @@ This option allows for passing in a job ID for the currently executing command.
 trying to track multiple commands across a single job. When the Hootenanny API database is used
 for job output storage, the ID will be stored in the job status table of the associated map.
 
+=== josm.map.cleaner.add.detail.tags
+
+* Data Type: bool
+* Default Value: `false`
+
+If true, cleaning/validation detail tags will be added to map outputs when cleaning maps with JOSM.
+
+=== josm.map.validator.java.implementation
+
+* Data Type: string
+* Default Value: `hoot/core/josm/JosmMapValidator`
+
+File path relative to Hootenanny root directory without file extension or prefixed namespace to
+Java class to use as the JOSM Java map validator/cleaner implementation.
+
+=== josm.max.elements.for.map.string
+
+* Data Type: int
+* Default Value: `2000000`
+
+The maximum size in elements for a map allowed to be passed as a string between Hootenanny and JOSM.
+The default value was determined experimentally and may need tweaking per environment.
+
+=== josm.validators.exclude
+
```

Comment:
Can configure which JOSM validators get run here with these two settings. Unfortunately, there's no way I've found yet to know whether a validator can do cleaning of features or not...its decided on a feature by feature basis. So, if you just want to do cleaning and no validation you could end up running validators that do no cleaning and just eat up processing time if you select all validators. Its up to the user to know which josm validators to run.